pymemcache/client/base.py

```diff
@@ -762,6 +762,31 @@ def _raise_errors(self, line, name):
             error = line[line.find(b' ') + 1:]
             raise MemcacheServerError(error)
 
+    def _value_helper(self, expect_cas, line, buf, remapped_keys,
+                      prefixed_keys):
+        """
+        This function is abstracted from _fetch_cmd to support different ways
+        of value extraction. In order to use this feature, _value_helper needs
+        to be overriden in the subclass.
+        """
+        if expect_cas:
+            _, key, flags, size, cas = line.split()
+        else:
+            try:
+                _, key, flags, size = line.split()
+            except Exception as e:
+                raise ValueError("Unable to parse line %s: %s"
+                                 % (line, str(e)))
```

Comment:
You can remove the `str()` around `e` because that will be done automatically by the `%s` string formatting token.

This will probably fit on one line as a result, too.

Reply:
sure sure, will change based on it.